ivadomed/scripts/curate_data_demo.py

```diff
@@ -0,0 +1,224 @@
+
+            # Check if the filename corresponds to the one in the derivatives dictionary
+            elif file in der:
+
+                # Derivatives go somewhere else
+                path_sub_id_dir_out = os.path.join(output_data, 'derivatives', 'labels', sub_id, 'microscopy')
+
+                # Define the output file path
+                path_file_out = os.path.join(path_sub_id_dir_out, sub_bids_full + der[file])
+            else:
+                # not a file we recognize
+                continue
+
+            # Create output subdirecotries and copy files to output
+            os.makedirs(os.path.dirname(path_file_out), exist_ok=True)
+            shutil.copyfile(path_file_in, path_file_out)
+
+    # Generate subject list
+    sub_list = sorted(d for d in os.listdir(output_data) if d.startswith("sub-"))
+
+    # Now that everything is curated, fill in the metadata
+    for sub_id in sub_list:
+        create_json_sidecar(output_data, sub_id)
+
+    # Create participants.tsv and samples.tsv
+    with open(output_data + '/samples.tsv', 'w') as samples, \
+            open(output_data + '/participants.tsv', 'w') as participants:
+        tsv_writer_samples = csv.writer(samples, delimiter='\t', lineterminator='\n')
+        tsv_writer_samples.writerow(["sample_id", "participant_id", "sample_type"])
+        tsv_writer_participants = csv.writer(participants, delimiter='\t', lineterminator='\n')
+        tsv_writer_participants.writerow(["participant_id", "species"])
+        for subject in sub_list:
+            row_sub = []
+            row_sub.append(subject)
+            row_sub.append('mus musculus')
+            tsv_writer_participants.writerow(row_sub)
+            subject_samples = sorted(glob.glob(os.path.join(output_data, subject, 'microscopy', '*.png')))
+            for file_sample in subject_samples:
+                row_sub_samples = []
+                row_sub_samples.append(os.path.basename(file_sample).split('_')[1])
+                row_sub_samples.append(subject)
+                row_sub_samples.append('tissue')
+                tsv_writer_samples.writerow(row_sub_samples)
+
+    # Create dataset_description.json
+    dataset_description = {"Name": "data_axondeepseg_tem",
+                           "BIDSVersion": "1.7.0",
+                           "License": "MIT"
+                           }
+
+    with open(output_data + '/dataset_description.json', 'w') as json_file:
+        json.dump(dataset_description, json_file, indent=4)
+
+    # Create dataset_description.json for derivatives/labels
+    dataset_description_derivatives = {"Name": "data_axondeepseg_tem labels",
+                                       "BIDSVersion": "1.7.0",
+                                       "PipelineDescription": {
+                                           "Name": "Axon and myelin manual segmentation labels"
+                                       }}
```

Comment:
```suggestion
    dataset_description_derivatives = {"Name": "demo dataset labels",
                                       "BIDSVersion": "1.7.0",
                                       "GeneratedBy": [{"Name": "Axon and myelin manual segmentation labels"}]
                                       }
```

The `PipelineDescription` field is deprecated in favor of `GeneratedBy`.